lib/sous-chef.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * This class is designed to parse the "What's cooking" mails sent to
+ * the Git mailing list about twice a week by the Git maintainer.
+ */
+
+export class SousChef {
+    public readonly mbox: string;
+    public readonly messageID: string | undefined;
+    public readonly branches: {
+        [branchName: string]: {
+            merged: string | undefined;
+            sectionName: string;
+            text: string;
+        }
+    };
+
+    public constructor(mbox: string) {
+        this.mbox = mbox;
+        this.branches = {};
+
+        const sections = mbox.split(/^-{10,}\n\[([^\]]+)\]\n/mg)
+        for (let i = 1; i < sections.length; i += 2) {
+            const sectionName = sections[i]
+
+            const branches = sections[i + 1].split(/\n\* ([a-z][^]+?)\n\n/m);
+            for (let j = 1; j < branches.length; j += 2) {
+                const match = branches[j]
+                    .match(/([^ ]+).*\n *(\(merged to [^)]+\))?/m);
+                if (!match) continue;
```

Comment:
Put `continue` in a block or are you okay with it when it is a one-liner?

Reply:
👍 

(I was just being lazy.)